ui/StatusQ/src/StatusQ/Core/Utils/ModelUtils.qml

```diff
@@ -2,6 +2,7 @@ pragma Singleton
 
 import QtQuick 2.14
 
+import StatusQ 0.1 as StatusQ
```

Comment:
Hmm... what is the goal here? :thinking:

Reply:
The tests were failing to import StatusQ.Internal and StatusQ was needed. I don't remember why I've added the alias though. Will try it out without it

Reply:
Ah right, I thought I'd seen this issue before; just wanted to make sure this fixes it :)